### ISSUE TITLE
Questions about the pre-trained weights on the SceneFlow dataset

### ISSUE THREAD

**Issue by lsyl20** [ZYangChen/MoCha-Stereo#32]
<img width="887" height="99" alt="Image" src="https://github.com/user-attachments/assets/586cc71b-48e4-4eab-afcb-29d94181da9d" />
Thank you for your excellent work. When I tested the provided pre-trained weights on the SceneFlow dataset, the reported performance(EPE=0.41) could only be reproduced on a subset of 400 test images. When I used the full test set, the results were not consistent with those reported in the paper.

**Comment by MichaelWangGo** [comment]
@lsyl20 Hi how could you load the weights? I got error

<img width="1302" height="87" alt="Image" src="https://github.com/user-attachments/assets/6105ef90-5387-466b-b3ad-6d80525e228c" />

**Comment by ZYangChen** [comment]
The value of 0.41 was indeed obtained by evaluating only the frames_finalpass subset. For this evaluation, the code was adapted from [this project](https://github.com/gangweix/IGEV/blob/main/IGEV-Stereo/evaluate_stereo.py).
